hack/generator/pkg/codegen/code_generator.go

```diff
@@ -39,12 +39,12 @@ type CodeGenerator struct {
 func NewCodeGenerator(configurationFile string) (*CodeGenerator, error) {
 	config, err := loadConfiguration(configurationFile)
 	if err != nil {
-		return nil, fmt.Errorf("failed to load configuration file '%v' (%w)", configurationFile, err)
+		return nil, errors.Wrapf(err, "failed to load configuration file '%v'", configurationFile)
```

Comment:
Should this change to use `%q` as well?
(Applies to others in this file as well as this one)

Reply:
Ooh, yup - I'll do another pass to make sure I've got all the `'%v'` and `'%s'` cases.